go.mod

```
@@ -1,6 +1,6 @@
 module github.com/NVIDIA/holodeck
 
-go 1.21.5
+go 1.22.0
```

Not that we don't want to do this, but what is the driver for this change? Do we depend on newer language features?

---

```suggestion
go 1.22
```
See https://go.dev/doc/modules/gomod-ref#go

---

Is not a required change, but a good to have one.